integration/js/utils/SdkBaseTest.js

```diff
@@ -14,6 +14,7 @@ class SdkBaseTest extends KiteBaseTest {
     this.baseUrl = this.url;
     if (testName === 'ContentShareOnlyAllowTwoTest') {
       this.url = this.getTransformedURL(this.url, 'max-content-share', 'true');
+      this.capabilities['extendedDebugging'] = false;
```

Comment:
Checking if this should be done for all or at least `ContentShare` tests as all are failing or do you plan to only test `ContentShareOnlyAllowTwoTest`?

Reply:
We might wanna disable all ContentShare tests, it could be difficult to confirm if it actually fixes the bug if only 'ContentShareOnlyAllowTwoTest' is disabled rather then all of them.

Reply:
ah, my bad- it should be for all.